chain/actors/builtin/verifreg/util.go

```diff
@@ -6,16 +6,18 @@ import (
 	"github.com/filecoin-project/go-state-types/big"
 	"github.com/filecoin-project/lotus/chain/actors"
 	"github.com/filecoin-project/lotus/chain/actors/adt"
+	builtin3 "github.com/filecoin-project/specs-actors/v3/actors/builtin"
 	"github.com/ipfs/go-cid"
 	"golang.org/x/xerrors"
 )
 
+// Assumes that the bitwidth for v3 HAMTs is the DefaultHamtBitwidth
 func getDataCap(store adt.Store, ver actors.Version, root cid.Cid, addr address.Address) (bool, abi.StoragePower, error) {
 	if addr.Protocol() != address.ID {
 		return false, big.Zero(), xerrors.Errorf("can only look up ID addresses")
 	}
 
-	vh, err := adt.AsMap(store, root, ver)
+	vh, err := adt.AsMap(store, root, ver, builtin3.DefaultHamtBitwidth)
```

Comment:
I'm thinking this `adt.AsMap` needs to go away and we need to handle the different map types in every place where it matters.

Reply:
Done.